.github/workflows/synctodevops.yml

```diff
@@ -0,0 +1,23 @@
+  
+name: Sync issue to Azure DevOps work item
+
+"on":
+  issues:
+    types:
+      [opened, edited, deleted, closed, reopened, labeled, unlabeled]
+
+jobs:
+  alert:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: danhellem/github-actions-issue-to-work-item@master
+        env:
+          ado_token: "${{ secrets.ADO_PERSONAL_ACCESS_TOKEN }}"
```

Comment:
>secrets. [](start = 26, length = 8)

Where do these secrets live?

Reply:
@anthonyvercolano they're in the repo settings, under the "Secrets" tab: https://github.com/Azure/azure-iot-sdk-node/settings/secrets